### PR TITLE
Add Flutter Edge Extensions release notes and update current sdk versions page.

### PR DESCRIPTION
**Pull request by cacheung**
Add Flutter Edge Extensions release notes and update current sdk versions page.

### CODE REVIEW

File: release-notes/README.md [hunk 1]
```diff
@@ -8,6 +8,20 @@ description: Release notes and change logs for the Adobe Experience Platform Mob
...
+
+* [Edge Network extension](https://pub.dev/packages/flutter_aepedge) is now supported in Flutter.
+
+### Flutter AEPEdgeIdentity 1.0.0
+
+* [Identity for Edge Network extension](https://pub.dev/packages/flutter_aepedgeidentity) is now supported in Flutter. 
+
+### Flutter AEPEdgeConsent 1.0.0
+
+* [Consent for Edge Network extension](https://pub.dev/packages/flutter_aepedgeconsent) is now supported in Flutter.
```
Comment (emdobrin) [inline release-notes/README.md:23]: let's group them together under a single header 
### AEP Flutter plugins for Edge Network
The following AEP Flutter plugins have been published:
- [flutter_aepedge@1.0.0](https://pub.dev/packages/flutter_aepedge)
- ...
- 
-e.g. https://aep-sdks.gitbook.io/docs/release-notes#aep-react-native-plugins

**Review by cdhoffmann — APPROVED** [review]

File: resources/upgrading-to-aep/current-sdk-versions.md [hunk 2]
```diff
@@ -171,6 +171,39 @@ For React Native, we recommend that you first install [Node.js](https://nodejs.o
...
 
+## Flutter (AEP 1.x) <a id="flutter-aep"></a>
+
+Adobe Experience Platform Mobile SDK plugin for Flutter supports Flutter **versions 2.0.0 or later**.
+
+{% hint style="danger" %}
+Adobe Experience Platform Mobile SDK plugins for Flutter are compatible with [Android](current-sdk-versions.md#android) and [iOS — Swift ](https://aep-sdks.gitbook.io/docs/resources/upgrading-to-aep/current-sdk-versions#ios-swift) libraries, at this time.
+{% endhint %}
+
+| Extension | pub.dev | Github | Sample App |
```
Comment (emdobrin) [inline resources/upgrading-to-aep/current-sdk-versions.md:182]: similar with the react native aep section, can we drop the github and sample app columns and add a mention add a mention under the header with the link to the flutter repo?

File: resources/upgrading-to-aep/current-sdk-versions.md [hunk 3]
```diff
@@ -171,6 +171,39 @@ For React Native, we recommend that you first install [Node.js](https://nodejs.o
...
+| [Edge](https://aep-sdks.gitbook.io/docs/foundation-extensions/experience-platform-extension) | [![pub package](https://img.shields.io/pub/v/flutter_aepedge.svg)](https://pub.dartlang.org/packages/flutter_aepedge) | [Link](https://github.com/adobe/aepsdk_flutter/tree/main/plugins/flutter_aepedge) | [Sample](https://github.com/adobe/aepsdk_flutter/tree/main/example) |
+| [Consent](https://aep-sdks.gitbook.io/docs/foundation-extensions/consent-for-edge-network) | [![pub package](https://img.shields.io/pub/v/flutter_aepedgeconsent.svg)](https://pub.dartlang.org/packages/flutter_aepedgeconsent) | [Link](https://github.com/adobe/aepsdk_flutter/tree/main/plugins/flutter_aepedgeconsent) | [Sample](https://github.com/adobe/aepsdk_flutter/tree/main/example) |
+| [EdgeIdentity](https://aep-sdks.gitbook.io/docs/foundation-extensions/identity-for-edge-network) | [![pub package](https://img.shields.io/pub/v/flutter_aepedgeidentity.svg)](https://pub.dartlang.org/packages/flutter_aepedgeidentity) | [Link](https://github.com/adobe/aepsdk_flutter/tree/main/plugins/flutter_aepedgeidentity) | [Sample](https://github.com/adobe/aepsdk_flutter/tree/main/example) |
```
Comment (emdobrin) [inline resources/upgrading-to-aep/current-sdk-versions.md:188]: can we maintain the order and name of the extensions the same
Mobile Core
Edge
EdgeIdentity
EdgeConsent
Assurance

File: resources/upgrading-to-aep/current-sdk-versions.md [hunk 4]
```diff
@@ -173,19 +173,22 @@ Beta versions of React Native Experience Platform Edge Network, Edge Identity, E
 
 ## Flutter (AEP 1.x) <a id="flutter-aep"></a>
 
+Adobe Experience Platform Mobile SDK plugin for Flutter supports Flutter **versions 2.0.0 or later**. For the latest installation instructions, see the `README` file in the [`aepsdk-flutter`](https://github.com/adobe/aepsdk_flutter#installation) repository.
+
 Adobe Experience Platform Mobile SDK plugin for Flutter supports Flutter **versions 2.0.0 or later**.
```
Comment (emdobrin) [inline resources/upgrading-to-aep/current-sdk-versions.md:178]: duplicated line, can be removed

**Review by emdobrin — APPROVED** [review]

**Pull request merged**